src/olympia/users/tests/test_views.py

```diff
@@ -812,6 +796,13 @@ def test_no_self_abuse(self):
         assert doc('#popup-staging #report-user-modal.modal').length == 0
         self.assertTemplateNotUsed(r, 'users/report_abuse.html')
 
+    def test_bio_xss(self):
+        self.user.update(biography='<script>alert("xss")</script>')
+        assert '<script>' in self.user.biography
+        r = self.client.get(self.url)
+        assert '<script>' not in r.content
+        assert '&lt;script&gt;' in r.content
```

Comment:
Just to be safe, could you add at least the `alert` part in the assertion? We're unlikely to ever add inline `<script>` tags to the page, but I wasn't 100% sure when I initially read it that it was testing the right thing because of that.